ui/src/institutions/utils.js

```diff
@@ -0,0 +1,43 @@
+import { List } from 'immutable';
+import { makeCompliantMetaDescription } from '../common/utils';
+
+export function getPapersQueryString(recordId) {
+  return `authors.affiliations.record.$ref:${recordId}`;
+}
+
+function getAddressText(address) {
+  const placeName = address.get('place_name');
+  const city = address.getIn(['cities', 0]);
+  const state = address.get('state');
+  const country = address.get('country');
+  const addressElements = [placeName, city, state, country];
+  const addressText = addressElements.filter(Boolean).join(', ');
+  return addressText;
+}
+
+export function getInstitutionMetaDescription(institution) {
+  const ITEM_SEPARATOR = ' and ';
```

Comment:
could be outside of the function

Reply:
why is it better to put it outside of the function? I see it as being specific to this function

Reply:
it's a constant that doesn't depend any of the function parameters.. so if you check `utils` we put these just above the function because ideally it's better to run them once instead of more.

For a string literal it doesn't matter rather than consistency, but for constants like `const VALUE = doSomeComputationToGetValue` or `const SUPER_NESTED_POSSIBLY_IMMUTABLEJS_VALUE = ...`, it might.